packages/server/modules/core/migrations/20220318121405_add_stream_favorites.js

```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+
+const TABLE_NAME = 'stream_favorites'
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+  await knex.schema.createTable(TABLE_NAME, (table) => {
+    table.string('streamId', 10).references('id').inTable('streams').onDelete('cascade')
+    table.string('userId', 10).references('id').inTable('users').onDelete('cascade')
+    table.timestamp('createdAt').defaultTo(knex.fn.now())
```

Comment:
pls switch to the same specific type implementation, that we use in comments (20220222173000_comments.js) for datetime types.

It solves the javascript vs postgres date storage precisison incompatibility